File: 1960-CheckIfTheSentenceIsPangram/1960-CheckIfTheSentenceIsPangram.cpp
// Last updated: 7/17/2026, 4:25:53 PM
class Solution {
public:
    bool checkIfPangram(string sentence) {
        string checkStr = "abcdefghijklmnopqrstuvwxyz";

        for (int i = 0; i < checkStr.size(); i++) {
            if (sentence.find(checkStr[i]) == string::npos) {
                return false;
            }
        }

        return true;
    }
};

File: 1960-CheckIfTheSentenceIsPangram/1960-CheckIfTheSentenceIsPangram.js
// Last updated: 7/17/2026, 4:25:56 PM
/**
 * @param {string} sentence
 * @return {boolean}
 */
var checkIfPangram = function(sentence) {
    let checkstr = "abcdefghijklmnopqrstuvwxyz";

    for(let i=0;i<checkstr.length;i++){
        if(!sentence.includes(checkstr[i])){
            return false;
        }
    }
    return true
};